install.sh

```
@@ -3,48 +3,123 @@
+              if ! apt-get purge -y pulseaudio; then
+                  echo "Failed to uninstall pulseaudio"
+                  exit 1
+              fi
+            fi
+            echo "Installing PipeWire..."
+            if ! apt-get install -y --no-install-recommends pipewire pipewire-alsa wireplumber; then
+                echo "Failed to install PipeWire"
+                exit 1
+            fi
+            # backup existing config
+            if [ -f /home/$OVOS_USER/.asoundrc]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)